docs/en/scanners.md

```diff
@@ -30,7 +30,8 @@ class MyKeyboard(KMKKeyboard):
             row_pins=self.row_pins,
             # optional arguments with defaults:
             columns_to_anodes=DiodeOrientation.COL2ROW,
-            interval=0.02,  # Debounce time in floating point seconds
+            interval=0.01, # How often the matrix is sampled
+            debounce_threshold=5, # Number of samples needed to change state
```

Comment:
I have a strong opinion on keeping the upstream default, even if it's "maybe suboptimal". Make a note in the docs that values around 5 are tested and should work.

Reply:
The way KMK is used we'd need this information in getting started... I'd prefer the minimum viable config to remain within its current scope.